site/core.py

```diff
@@ -0,0 +1,130 @@
+'''operacoes do banco de dados'''
+from sqlalchemy import (create_ENGINE, MetaData, Column,

```

Comment:
Aqui tem um errinho de nome

```suggestion
from sqlalchemy import (create_engine, MetaData, Column,

```